calibre-web.json

```
@@ -0,0 +1,59 @@
+{
+  "Calibre Web": {
+    "description": "Calibre-Web is a web app providing a clean interface for browsing, reading and downloading eBooks using an existing Calibre database.<p>The Web interface allows user logins and the management and download of e-Books through a clean web interface.</p><p>For more information please see <a href='https://github.com/janeczku/calibre-web' target='_blank' rel='noopener noreferrer'>the oficial Github page</a> and <a href='https://hub.docker.com/r/linuxserver/calibre-web' target='_blank' rel='noopener noreferrer'>the Docker container page.</a></p></br>If linking to the Calibre Rockon use the directory '/config/Calibre Library' as your database location during configuration in the WebUI.",
+    "more_info": "",
+    "version": "0.0.1",
+    "website": "https://github.com/janeczku/calibre-web",
+    "ui": {
+      "slug": ""
+    },
+    "volume_add_support": true,
+    "containers": {
+      "calibreWeb": {
+        "image": "linuxserver/calibre-web",
+        "launch_order": 1,
+        "opts": [
+          [
+            "-v",
+            "/config",
+            "-v",
+            "/books"
+          ]
+        ],
+        "ports": {
+          "8083": {
+            "description": "Calibre-web UI port.",
+            "host_default": 8083,
+            "label": "WebUI port",
+            "protocol": "tcp",
+            "ui": true
+          }
+        },
+        "volumes": {
+          "/config": {
+            "description": "Choose the Share where you store Calibre configuration data. Eg: The Share called calibre-config.<p>If using the Calibre rockon, this must be the configuration share of Calibre.  When configuring Calibre-Web, use '/config/Calibre Library' as the database location.</p>",
+            "label": "Config Storage"
+          },
+          "/books": {
+            "description": "Choose the Share with e-Book content. Eg: Select the share where Calibre reads ebooks from. It will be available as /media. You can also add more shares later.<p>If using the Calibre Rockon, you must select the same shares that were used when configuring that rockon or the books will not be accessible.</p>",
+            "label": "Media Storage"
+          }
+        },
+        "environment": {
+          "PUID": {
+            "description": "Enter a valid UID of an existing user with permission to media shares to run Calibre as.",
+            "label": "UID",
+            "index": 1,
+            "default": 1000
+          },
+          "PGID": {
+            "description": "Enter a valid GID of an existing user with permission to media shares to run Calibre as.",
+            "label": "GID",
+            "index": 2,
+            "default": 1000
```

Defaults to `100` in openSUSE:
```suggestion
            "default": 100
```